packages/react-router/src/context.ts

```diff
@@ -7,4 +7,6 @@ export const RouterContext = createContext<import('./router').Router | null>(
 );
 export const SwitchContext = createContext<{matched(): void} | null>(null);
 
-export const FocusContext = createContext<RefObject<Focusable> | null>(null);
+export const FocusContext = createContext<MutableRefObject<
+  Focusable | undefined
```

Comment:
I think it will be helpful to allow `undefined | null` here, since we end up using both to represent "empty" in various places.